Dockerfile

```diff
@@ -17,4 +17,6 @@ ENV FLASK_DEBUG=true
 ENTRYPOINT poetry run flask run --host 0.0.0.0
 # configure for local development
 
-
+FROM base as test
+Copy .env.test /app/
```

Comment:
Although this works fine, it's standard to use uppercase for the Dockerfile instruction, i.e.
```
COPY .env.test /app/
```

While I'd argue that  this doesn't really matter if a team chooses to ignore it, it is worth being consistent with the other code around, and you've always used capitals for these commands elsewhere in the file!